src/main/java/nl/tudelft/lifetiles/sequence/controller/SequenceController.java

```diff
@@ -131,8 +128,8 @@ private void repaint() {
         ObservableList<Label> sequenceItems = FXCollections
                 .observableArrayList();
         for (final Sequence sequence : sequences.values()) {
-            String id = sequence.getIdentifier();
-            Label label = new Label(id);
+            String ident = sequence.getIdentifier();
```

Comment:
Making variable names longer should yield more meaning, *ident* is not quite clear; `identifier` would be clearer.